packages/SwingSet/test/test-vat-imports.js

```diff
@@ -9,8 +9,13 @@ async function requireHarden(t, withSES) {
 }
 
 test('vat can require harden with SES', async t => {
-  await requireHarden(t, true);
-  t.end();
+  try {
+    await requireHarden(t, true);
+  } catch (e) {
+    t.isNot(e, e, 'unexpected exception');
```

Comment:
why not use `t.fail`?

Reply:
`t.fail(e)` does not print the exception stack trace.